### DISCUSSION TITLE
7194 - Deployment com sucesso, mas apresenta erro ao acessar o link do azure

### DISCUSSION THREAD

**Discussion by rafaelcmpereira** [balta-io/7194#7] [Perguntas e Respostas]
Não encontrei nenhum erro aparente, pois da build normal e local funciona, mas no azure só apresenta esse erro...

:( Application Error
If you are the application administrator, you can access the diagnostic resources.

https://github.com/rafaelcmpereira/balta-7194-deploy

**Comment by andrebaltieri** [comment]
Bom dia @rafaelcmpereira, como vai?
Tenta ver se este link ajuda: https://balta.io/blog/aspnet-deploy-github-actions-azure
